src/main/java/com/stripe/model/Person.java

```diff
@@ -200,10 +200,6 @@ public Person delete(Map<String, Object> params, RequestOptions options) throws
     @SerializedName("director")
     Boolean director;
 
-    /** Whether the person has a significant control of the account’s legal entity. */
-    @SerializedName("executive")
-    Boolean executive;
```

Comment:
`executive` field in now hidden in OpenAPI spec. I'll make a change in pay-server to add it back..
Put it in TODO here
https://git.corp.stripe.com/stripe-internal/sdk-autogen-java/blob/8eac5b0e7eedeedbfaff8980c6b0005d24ef961e/generated/diff_change_log_fields.md#todo-add-back-from-pay-server

Reply:
It's going away, please keep it removed! (also we should avoid linking to internal links on public repos!)

Reply:
ah got it! will take no action on this.